### PR TITLE
"add lazaro.py from branch version-1"

### PR DESCRIPTION
**Pull request by Lazaro-Franco**
Please check any errors in my code

### CODE REVIEW

File: Lazaro.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+print("1-Basic Info")
+print("2-Goals ")
+print("3-Comments from teammates")
+print("4-Exit")
+user_input =  int(input("Enter your choice: "))
+
+if user_input == 1:
+    print("Age: 69")
+    print("Birthday: April 11, 2005")
+    print("Hobby: Watching anime, reading manga, Drawing")
```
Comment (arroyo-johnmatthew) [inline Lazaro.py]: "Drawing" should be "drawing"

File: Lazaro.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    print("Favorite Food: Okonomiyaki")
+    print("Favorite car: Citroen 2cv, Fiat nuova 500")
+    print("Favorite Anime: Lupin IV, Assassination Classroom")
+    
+elif user_input == 2:
+   print("==== Goals ====") 
+   print("To become an artist or game developer")
+   print("Create my own animated series")
+   print("To become a Great animator")
```
Comment (arroyo-johnmatthew) [inline Lazaro.py]: "Great" should be "great"

**Review by arroyo-johnmatthew — CHANGES_REQUESTED** [review]
Please comply to all of these changes. Thank you.

File: Lazaro.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
+print("Greetings: Franco Alfonso C. Lazaro")
+print("1-Basic Info")
+print("2-Goals ")
+print("3-Comments from teammates")
+print("4-Exit")
+user_input =  int(input("Enter your choice: "))
+
+if user_input == 1:
+    print("Age: 69")
```
Comment (nolluda-johncarlo) [inline Lazaro.py]: Wrong age . Put your actual age

**Review by nolluda-johncarlo — CHANGES_REQUESTED** [review]
Please fix the code based on my observations.

**Review by arroyo-johnmatthew — APPROVED** [review]
It looks good. A job well done.

**Review by nolluda-johncarlo — APPROVED** [review]
Good job!

**Pull request merged**